blazor/file-manager/upload.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: Upload in Blazor FileManager Component | Syncfusion
+description: Checkout and learn here all about available Upload in Syncfusion Blazor FileManager component and much more.
+platform: Blazor
+control: File Manager
+documentation: ug
+---
+
+# Upload in Blazor FileManager component
+
+The Blazor FileManager component provides a [FileManagerUploadSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerUploadSettings.html) feature that allows users to upload files to the FileManager. It provides various settings and options to customize how files are uploaded, such as controlling file size, restricting file types, and enabling chunk uploads.
+
+## Directory Upload
+
+The [DirectoryUpload](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerUploadSettings.html#Syncfusion_Blazor_FileManager_FileManagerUploadSettings_DirectoryUpload) property controls whether users can browse and upload entire directories (folders) in the Syncfusion Blazor FileManager component. 
+
+To enable directory upload, set the `DirectoryUpload` property to `true` in the `FileManagerUploadSettings` configuration.
+
+When set to `true`, this property enables directory upload in the FileManager, allowing users to upload entire folders. If set to `false`, only individual files can be uploaded.
+
+```cshtml
+
+@using Syncfusion.Blazor.FileManager
+
+<SfFileManager TValue="FileManagerDirectoryContent">
+    <FileManagerUploadSettings DirectoryUpload  = "true"></FileManagerUploadSettings>
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api//FileManager/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/test/FileManager/GetImage">
+    </FileManagerAjaxSettings>
+</SfFileManager>
+
+```
+
```

Comment:
Add note on file and folder cannot be uploaded at the same time

Reply:
added  note on directory upload